Makefile

```diff
@@ -139,28 +136,29 @@ endif
 .EXPORT_ALL_VARIABLES:
 
 
-.PHONY: image_build run destroy start_minikube delete_minikube run destroy deploy_assisted_service deploy_assisted_operator create_environment delete_all_virsh_resources _deploy_assisted_service _deploy_nodes  _destroy_terraform
+.PHONY: image_build run destroy start_minikube delete_minikube deploy_assisted_service deploy_assisted_operator delete_all_virsh_resources _deploy_assisted_service _deploy_nodes _destroy_terraform
 
 ###########
 # General #
 ###########
 
-all: create_full_environment run_full_flow_with_install
-
+all: setup run_full_flow_with_install
 
-destroy: destroy_nodes kill_port_forwardings destroy_onprem stop_load_balancer
+destroy: destroy_nodes delete_minikube kill_port_forwardings destroy_onprem stop_load_balancer
 
 ###############
 # Environment #
 ###############
-create_full_environment:
+setup:
```

Comment:
@lalon4 will that affect QE or automations?

Reply:
We have ``create_full_environment: setup`` for backup
But yes, it will be a good idea to dry-run it as part of QE's workflows

Reply:
We dont use create_full_environment script in our jenkins (we dont need it & we dont want to impact our CI machines with more packages to install)
BUT we need the LB so we use: ./scripts/install_environment.sh config_nginx
https://gitlab.cee.redhat.com/ocp-edge-qe/ocp-edge/-/blob/master/linchpin-workspace/hooks/ansible/ocp-edge-setup/roles/assisted-installer/tasks/assisted-run-functional-api-tests.yaml#L188

Reply:
okay
``install_environment.sh`` script hasn't changed in this PR, so I guess we should be okay on that front